packages/thumbprint-global-css/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@thumbtack/thumbprint-global-css",
+    "description": "Global reset, body, and anchor styling for building Thumbtack UI",
+    "version": "0.1.0",
+    "author": "Tom Genoni <tom@thumbtack.com>",
+    "main": "dist/thumbprint-global.css",
+    "style": "dist/thumbprint-global.css",
+    "files": [
+        "dist"
+    ],
+    "publishConfig": {
+        "registry": "https://registry.npmjs.org/",
+        "access": "public"
+    },
+    "license": "Apache-2.0",
+    "scripts": {
+        "start": "yarn clear && yarn generateCss",
+        "clear": "rm -rf dist",
+        "generateCss": "mkdir -p dist && node-sass -q src/thumbprint-global.scss dist/thumbprint-global.css && postcss dist/thumbprint-global.css --map --use autoprefixer css-mqpacker cssnano --replace",
```

Comment:
Is `css-mqpacker` needed? I think that is left-over from Atomic.

Reply:
Correct. Will remove.